app/main.py

```diff
@@ -1,44 +1,48 @@
+import asyncio
 import time
 
 from iot.devices import HueLightDevice, SmartSpeakerDevice, SmartToiletDevice
 from iot.message import Message, MessageType
 from iot.service import IOTService
 
+from utils import run_parallel, run_sequence
```

Comment:
Ensure that the `utils` module is correctly imported. The import statement should be `from iot.utils import run_parallel, run_sequence` if `utils.py` is located in the `iot` package.